docs/TektonConfig.md

```diff
@@ -278,6 +301,40 @@ spec:
       default-tekton-hub-catalog: tekton
 ```
 
+### OpenShiftPipelinesAsCode
+PipelinesAsCode section allows user to customize the Pipelines as Code features. This allow user to customize the values in configmaps.
```

Comment:
"The" at the start - grammar, also "allows the user" or "allows you" (both times). Also "to customize values in configmaps" does not sound clear. I suggest "When you change the TektonConfig CR, he Operator automatically applies the settings to custom resources and configmaps in your installation"